docs/sql-guide/functions/function-setcontains.md

```diff
@@ -12,15 +12,15 @@ SETCONTAINS() tests membership of a value within a set expression.
 ## Syntax
 
 ```
-setcontains(set, value)
+setcontains({set, value})
 ```
 
 ## Arguments
 
-| Argument | Description | Data type | Return value |
-|---|---|---|---|
-| `set` | The set in which value is being tested for membership. | `stringset` or `idset` |
-| `value` | The value to test membership for in the set. `value` must be assignment compatible with the element type of the set. |
+| Argument | Description | Required | Data type |
```

Comment:
added `Required` column to back-up the change to the syntax

Reply:
Ya we need standards on this stuff. Data type in a different spot here: https://docs.featurebase.com/docs/sql-guide/functions/function-prefix/. Further information on some. Let's not solve this hear. This is stuff on the top of my list post re-writes.